internals/maintenance_scripts.py

```diff
@@ -803,6 +804,41 @@ def calc_needs_work_started_on(
                if v.state == Vote.NEEDS_WORK)
 
 
+class FetchWebdxFeatureId(FlaskHandler):
+
+  def get_template_data(self, **kwargs) -> str:
+    """Fetch the complete list of Webdx feature ID available from
+    webstatus.dev APIs and store them in datastore.
+    """
+    self.require_cron_header()
+
+    if settings.UNIT_TEST_MODE:
```

Comment:
You should move the webstatus_dev_url into [settings.py](https://github.com/GoogleChrome/chromium-dashboard/blob/main/settings.py)

There is existing logic to toggle settings based on UNIT_TEST_MODE.

```diff
+API_WEBSTATUS_DEV_URL = 'https://api.webstatus.dev'

if UNIT_TEST_MODE:
  APP_TITLE = 'Local testing'
  SITE_URL = 'http://127.0.0.1:7777/'
+  API_WEBSTATUS_DEV_URL = 'https://api.server.test'
```

Then here, you could just do:

```python
client = DefaultApi(ApiClient(Configuration(settings.API_WEBSTATUS_DEV_URL)))
```